pom.xml

```diff
@@ -437,6 +437,13 @@
 			<artifactId>poi-ooxml</artifactId>
 			<scope>provided</scope>
 		</dependency>
+        <!-- InputPhone dependencies -->
+        <dependency>
+            <groupId>com.googlecode.libphonenumber</groupId>
+            <artifactId>libphonenumber</artifactId>
+            <version>8.10.7</version>
+            <scope>provided</scope>
```

Comment:
So, I added this dependency as provided. I'm not sure if we can just add it (Apache 2, 340kB)

Reply:
If its Apache 2 we can add it.